adage/adage/settings.py

```diff
@@ -0,0 +1,147 @@
+"""
+Django settings for adage project.
+
+Generated by 'django-admin startproject' using Django 2.2.5.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.2/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.2/ref/settings/
+"""
+
+import os
+import yaml
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+
+# Read secrets from YAML file
+path = os.path.join(BASE_DIR, 'adage', 'config.yml')
+with open(path) as read_file:
+    config = yaml.full_load(read_file)
+
+# Quick-start development settings - unsuitable for production
+# See https://docs.djangoproject.com/en/2.2/howto/deployment/checklist/
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = config.get('secret_key', 'django_secret_key')
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = config.get('debug', True)
```

Comment:
Good point. In the default `settings.py` (generated by `django-admin startproject` command), this section was:
```
# SECURITY WARNING: don't run with debug turned on in production!
DEBUG = True
```
which is probably where the lines in `adage` came from. I found it a little annoying when deploying the production server, because I had to modify `settings.py` directly. That is why I put this option in `config.yml` now. 

This is also the reason why the default is `True` here, because I try to keep it consistent with the original setting.

Reply:
I'm totally fine with defaulting it to `True` for now. It will be a while before we come to the point of deploying this in production. At the same time, that time delay is why I was thinking it might be good to change the default now so we don't forget. After the code review is done we tend to not revisit questions like this.

Is there a place for us to keep notes about things we want to remember to do before production deployment? If you prefer to leave the development default for now then maybe we should have a list of reminders about this and other things we know we need to come back to.

Reply:
@mhuyck, I created issue #5 to keep track of the config of production server. Please feel free to comment on it.